python/build-wheel-macos.sh

```diff
@@ -16,19 +16,21 @@ DOWNLOAD_DIR=python_downloads
 NODE_VERSION="14"
 PY_VERSIONS=("3.6.1"
              "3.7.0"
-             "3.8.2")
+             "3.8.2"
+             "3.9.1")
 PY_INSTS=("python-3.6.1-macosx10.6.pkg"
           "python-3.7.0-macosx10.6.pkg"
-          "python-3.8.2-macosx10.9.pkg")
+          "python-3.8.2-macosx10.9.pkg"
+          "python-3.9.1-macosx10.9.pkg")
 PY_MMS=("3.6"
         "3.7"
-        "3.8")
+        "3.8"
+        "3.9")
 
-# The minimum supported numpy version is 1.14, see
-# https://issues.apache.org/jira/browse/ARROW-3141
 NUMPY_VERSIONS=("1.14.5"
                 "1.14.5"
-                "1.14.5")
+                "1.14.5"
+                "1.19.3")
```

Comment:
Have we tried `1.14.5` or some other version earlier than `1.19` here? I know the numpy docs state that `1.19` is the first version the officially supports Python 3.9, but if it happens to also work with an older version it would be preferable to put that here since that would mean Ray wheels on 3.9 work also with older numpy versions, to the degree that Python 3.9 supports them. For 3.7 and 3.8 things were more compatible than expected here, maybe that's also the case for 3.9? It's worth trying it out.

Thanks for all the contributions to make Ray work on 3.9, that's terrific!

Reply:
Actually, apache arrow is doing the same, so I think it is ok: https://github.com/apache/arrow/blob/master/python/requirements-wheel-build.txt#L3

Reply:
Sweet, then I'll keep it as it is.